github-action.py

```diff
@@ -920,17 +920,25 @@ def build(self):
                 )
 
                 additional_info = None
-                server = self.config.get("openqa", {}).get("server", "openqa.qubes-os.org")
-                key = self.config.get("openqa", {}).get("key", None)
-                secret = self.config.get("openqa", {}).get("secret", None)
+                server = (
+                    self.config.get("github", {})
+                    .get("openqa", {})
+                    .get("server", "openqa.qubes-os.org")
+                )
+                key = self.config.get("github", {}).get("openqa", {}).get("key", None)
+                secret = (
+                    self.config.get("github", {}).get("openqa", {}).get("secret", None)
+                )
                 if all([key, secret, OpenQA_Client, OpenQAClientError]):
                     openqa_client_path = Path.home() / ".config/openqa/client.conf"
                     openqa_client_path.parent.mkdir(parents=True, exist_ok=True)
                     with open(openqa_client_path, "w") as f:
-                        f.write(f"""[{server}]
+                        f.write(
+                            f"""[{server}]
 key = {key}
 secret = {secret}
-""")
+"""
+                        )
```

Comment:
Looking at it more, shouldn't qubes-infrastructure simply write this file instead?

Reply:
Yes you are right, also, I would adapt the check with `openqa_client_path.exists()`.